### PR TITLE
Make the README more informative with a focus on the project itself

### PR DESCRIPTION
**Pull request by Ingvarstep**
Hi @urchade ,

Because the current README highlights other commercial tools and projects, it creates the following issues:
- The README doesn't inform well about the key features and benefits of the GLiNER project;
- It creates questions about the maintainability of the project and its open-source goals;
- The README doesn't clearly describe the contribution guidelines.

I have crafted a new README proposal to address the issues described above and to make it more informative and better capture the GLiNER project itself.

To highlight GLiNER2 and other amazing projects from the Ecosystem, I created a dedicated section listing them. 

Looking forward to your suggestions and feedback.

@tomaarsen, it would be awesome to get your review and feedback as well.

Best,
Ihor

### CODE REVIEW

File: assets/banner.png
Comment (tomaarsen) [inline assets/banner.png:1]: Ooh, looks very clean

File: README.md [hunk 1]
```diff
@@ -1,90 +1,110 @@
...
-## Example Notebooks
+---
 
-Explore various examples including finetuning, ONNX conversion, and synthetic data generation. 
+## Why GLiNER?
+
+<table>
+<tr>
+<td width="33%" align="center">
+<h3>Zero-shot Recognition</h3>
```
Comment (tomaarsen) [inline README.md]:
```suggestion
<b>Zero-shot Recognition</b>
```
GitHub will turn h3 headers into clickable headers that users can link to, e.g. https://github.com/urchade/GLiNER/blob/3f6dd3eefce4a6ff3f66cda9b6c0a1eebb7756a0/README.md#zero-shot-recognition
Perhaps that's not ideal, and just bold is nicer? (perhaps a `<br>` is also required)

File: README.md [hunk 2]
```diff
@@ -1,90 +1,110 @@
...
+<h3>Fine-Tune in Minutes</h3>
+<p>Few-shot learning on small datasets — bring your own labels and get competitive results fast</p>
+</td>
+</tr>
+</table>
 
-- [Example Notebooks](https://github.com/urchade/GLiNER/tree/main/examples)
-- Finetune on Colab &nbsp;[<img align="center" src="https://colab.research.google.com/assets/colab-badge.svg" />](https://colab.research.google.com/drive/1HNKd74cmfS9tGvWrKeIjSxBt01QQS7bq?usp=sharing)
-## 🛠 Installation & Usage
+---
```
Comment (tomaarsen) [inline README.md]: This is very minor, but I think the --- lines are not needed when you also have a new header starting below each

File: README.md [hunk 3]
```diff
@@ -1,90 +1,110 @@
...
+
+## Quick Start
```
Comment (tomaarsen) [inline README.md:69]: I like the updated README here, very bite-sized. All that's missing (I think) is a link to docs below each (sub)section, e.g. below the `Output:`, the `Quanitzation and Compilation`, the `Serving`, etc.). You already have some links below Training, that's good.

File: README.md [hunk 4]
```diff
@@ -93,48 +113,126 @@ Use `quantize=True` and `compile_torch_model=True` for up to ~1.9x faster GPU in
...
-    quantize=True,            # or "fp16", "bf16"
+    quantize=True,
     compile_torch_model=True,
 )
 ```
 
-Or apply after loading:
+## Serving
+
+GLiNER provides a built-in serving interface for batch inference:
```
Comment (tomaarsen) [inline README.md]: I would add 1 extra sentence to explain why someone would serve instead of using the script above.

File: README.md [hunk 5]
```diff
@@ -93,48 +113,126 @@ Use `quantize=True` and `compile_torch_model=True` for up to ~1.9x faster GPU in
...
 ```python
 model = GLiNER.from_pretrained(
-    "urchade/gliner_medium-v2.1",
+    "gliner-community/gliner_small-v2.5",
     map_location="cuda",
-    quantize=True,            # or "fp16", "bf16"
+    quantize=True,
     compile_torch_model=True,
 )
 ```
```
Comment (tomaarsen) [inline README.md:121]: I'm missing the quantization/compilation gains a bit. As a user I'd like to know what this does for me.

File: README.md [hunk 6]
```diff
@@ -213,4 +339,10 @@ F.initiatives has been an expert in public funding strategies for R&D, Innovatio
   <img src="logo/FI_COMPLET_CW.png" alt="FI Group" width="300"/>
 </p>
 
-We also extend our heartfelt gratitude to the open-source community for their invaluable contributions, which have been instrumental in the success of this project.
+We also extend our heartfelt gratitude to the open-source community for their invaluable contributions, which have been instrumental in the success of this project. ❤️
+
+---
+
+<div align="center">
```
Comment (tomaarsen) [inline README.md:334]: Not very crucial in my opinion, but it's fine to keep.

File: README.md (hunk 4 above)
Comment (tomaarsen) [reply README.md]: Btw, is this merged? I see #346 which seems to refer to this

File: README.md [hunk 7]
```diff
@@ -93,48 +113,126 @@ Use `quantize=True` and `compile_torch_model=True` for up to ~1.9x faster GPU in
...
-| + compile | 0.8107 | 1.31x |
-| **+ quantize + compile** | **0.8107** | **1.94x** |
+GLiNER supports multiple architectures tailored to different use cases:
 
-**Quantization options:**
-- `quantize=True` or `quantize="fp16"` — float16 half-precision. Best GPU speedup (~1.35x).
-- `quantize="bf16"` — bfloat16. Better numerical stability, slightly less speedup (~1.2x).
-- `quantize="int8"` — int8 quantization. On CPU, uses built-in FBGEMM int8 kernels (~1.6x speedup). On GPU, uses [torchao](https://github.com/pytorch/ao) int8 weight-only quantization (~50% memory reduction, no speed gain). Intended for models fine-tuned with quantization-aware training (QAT). Stock DeBERTa-based models lose accuracy with int8.
-- On CPU, fp16/bf16 quantization reduces memory usage but does not improve speed.
+| Architecture | Description |
```
Comment (tomaarsen) [inline README.md]: Nice to see this, I honestly lost track and I've been following GLiNER-related developments pretty close

File: README.md [hunk 8]
```diff
@@ -93,48 +113,126 @@ Use `quantize=True` and `compile_torch_model=True` for up to ~1.9x faster GPU in
...
+|---|---|
+| **Uni-encoder** | Strong zero-shot capabilities, supports up to ~50 entity types. The original GLiNER architecture. |
+| **Bi-encoder** | Scalable to massive numbers of entity types via separate text and label encoding. |
+| **RelEx** | Joint NER and relation extraction in a single model. |
+| **GLiNER Decoder** | Hybrid architecture for open NER — entity types are generated with a small decoder for maximum flexibility. |
```
Comment (tomaarsen) [inline README.md]:
```suggestion
| **GLiNER Decoder** | Hybrid architecture for open NER: entity types are generated with a small decoder for maximum flexibility. |
```

Personal preference: I would remove the em dashes throughout

File: README.md [hunk 9]
```diff
@@ -93,48 +113,126 @@ Use `quantize=True` and `compile_torch_model=True` for up to ~1.9x faster GPU in
...
 
-**Compilation notes:**
-- `compile_torch_model=True` uses [torch.compile](https://pytorch.org/docs/stable/torch.compiler.html) which JIT-compiles the model via [Triton](https://github.com/triton-lang/triton) kernels. The first inference call will be slower due to compilation, but all subsequent calls benefit from the compiled graph. This is only available on **Linux and WSL** (not native Windows or macOS).
+For more details, see the [documentation](https://urchade.github.io/GLiNER).
+
+---
+
+## Popular Use Cases
```
Comment (tomaarsen) [inline README.md:191]: Even better if you can immediately link from a use case to a matching example like these: https://urchade.github.io/GLiNER/usage.html#example-1-extract-company-information

**Review by tomaarsen — COMMENTED** [review]
Sure, I can give a review 🤗 

Overall some nice extensions I think, and a pretty logical structure, i.e.: title, tags, relevant banner, one paragraph, and then straight into benefits followed by a very solid quick start. I would probably clean up the em dashes though. In my eyes, they read as low-effort, when clearly a lot of love has gone into this project.

Edit: I would more clearly link to the existing models, e.g.: https://huggingface.co/models?library=gliner
I realise now that I miss that a bit.

File: assets/banner.png
Comment (Ingvarstep) [reply assets/banner.png:1]: It looks minimalistic, but it was actually hard to find the right prompt to generate it that way ;)

File: README.md (hunk 1 above)
Comment (Ingvarstep) [reply README.md]: I think just bold will be good.

File: README.md (hunk 2 above)
Comment (Ingvarstep) [reply README.md]: Actually, I looked into it a bit more, and we can definitely remove the --- lines when there is a new header after them.

File: README.md (hunk 4 above)
Comment (Ingvarstep) [reply README.md]: It's still under review, but API and documentation should be stable.

File: README.md (hunk 9 above)
Comment (Ingvarstep) [reply README.md:191]: This is a good point, I am adding cross-links to the documentation with real examples.

**Comment by Ingvarstep** [comment]
Hi @urchade , any thoughts on the readme update?
